inventory.py

```diff
@@ -38,25 +52,26 @@ def parse_params():
         os.environ['ATLAS'] = options.atlas
     if options.sleep:
         os.environ['SLEEP'] = options.sleep
-    else:
+    if not options.sleep:
         os.environ['SLEEP'] = "1800"
```

Comment:
Put in an info-message for `defaulting sleep to 1800s`

Reply:
Alright.